app/controllers/spaces_controller.rb

```diff
@@ -1,88 +1,64 @@
 # frozen_string_literal: true
 
 class SpacesController < ApplicationController
-  before_action :set_space, only: %i[show edit update destroy]
+  before_action :set_space, only: %i[ show edit update destroy ]
   before_action :check_multi_tenant_mode, only: %i[new index]
-  before_action :set_space_and_user_roles, only: %i[create]
 
-  # GET /spaces or /spaces.json
   def index
     @spaces = current_user.spaces.page params[:page]
   end
 
-  # GET /spaces/1 or /spaces/1.json
-  def show; end
+  def show
+  end
 
-  # GET /spaces/new
   def new
     @space = Space.new
   end
 
-  # GET /spaces/1/edit
-  def edit; end
+  def edit
+  end
 
-  # POST /spaces or /spaces.json
   def create
-    respond_to do |format|
-      if @space.save
-        format.html { redirect_to space_url(@space), notice: "Space was successfully created." }
-        format.json { render :show, status: :created, location: @space }
-      else
-        format.html { render :new, status: :unprocessable_entity }
-        format.json { render json: @space.errors, status: :unprocessable_entity }
```

Comment:
note: these json responders aren't working as expected, makes the decision to remove them easy now 👍